### PR TITLE
Update sponsors

### PR DESCRIPTION
**Pull request by anthonyj33**
.xyz, Interview Cake, Give My Certificate, JDoodle

~~Need to verify links for Give My Certificate & JDoodle~~ Nvm its just a link to homepage so thats taken care of

### CODE REVIEW

**Comment by Awesome-E** [comment]
1. Bruh
image

2. I already told you this, but we have to make the 3.0 previous event before starting this because that way you can copy everything over, then remove them all instead of removing things, adding things, and not knowing which one is updated or not

**Review by Awesome-E — CHANGES_REQUESTED** [review]
image is broken (see above)

**Comment by anthonyj33** [comment]
should be sorted out

File: views/index.html.hbs [hunk 1]
```diff
@@ -266,112 +266,28 @@
...
-        {{#> sponsor }}
-          {{#*inline "name"}}DevITjobs{{/inline}}
-          {{#*inline "image"}}devit.webp{{/inline}}
-          {{#*inline "link"}}https://devitjobs.uk/startup-jobs{{/inline}}
           {{#*inline "tier"}}bronze{{/inline}}
         {{/sponsor}}
         {{#> sponsor }}
           {{#*inline "name"}}Gen XYZ{{/inline}}
-          {{#*inline "image"}}xyz.webp{{/inline}}
+          {{#*inline "image"}}xyz.webp{{/inline}}
```
Comment (Awesome-E) [inline views/index.html.hbs:277]: No space at end of line

**Review by Awesome-E — CHANGES_REQUESTED** [review]
Almost!

Two small things:
1. The comment
2. Alphabetical order for sponsors within the same tier. I know it's something minor and easy to miss, but attention to detail pays off in the long run – plus that gives some sense to how it "should be" rather than just doing ordering randomly.

**Review by Awesome-E — APPROVED** [review]

**Pull request merged**